src/ViewRenderer/ApplicationViewInjection.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ViewRenderer;
+
+use Yiisoft\Router\UrlMatcherInterface;
+use Yiisoft\Yii\Web\User\User;
+
+class ApplicationViewInjection implements
+    LayoutParamsInjectionInterface,
+    MetaTagsInjectionInterface,
+    LinkTagsInjectionInterface
+{
+
+    private User $user;
+    private UrlMatcherInterface $urlMatcher;
+
+    public function __construct(
+        User $user,
+        UrlMatcherInterface $urlMatcher
+    ) {
+        $this->user = $user;
+        $this->urlMatcher = $urlMatcher;
+    }
+
+    public function getLayoutParams(): array
+    {
+        return [
+            'user' => $this->user->getIdentity(),
```

Comment:
https://github.com/yiisoft/app/blob/master/config/params.php#L83

Reply:
In defaultParameters set to both: content and layout.  
If need only for layout defaultParameters not suitable.

Reply:
I've meant that such configuration is way more exposed to end user and simpler than modifying ApplicationViewInjection itself. It could be generalized and made configurable.